front/components/items/Appointment/helper.ts

```diff
@@ -28,3 +28,22 @@ export const getDateOfAppointments = ({ appointments }: props) => {
         }
     });
 };
+
+export const getIdOfAppointment = ({
+    fullDataAppointment,
+    chosenTherapist,
+    chosenDate,
+    chosenTime,
+}: propsOfGetId) => {
+    return fullDataAppointment?.filter((item) => {
+        if (item.attributes.therapist.data.id === chosenTherapist) {
+            if (chosenDate !== undefined) {
+                if (item.attributes.date === formingDate({ chosenDate })) {
+                    if (item.attributes.time === chosenTime + ':00.000') {
```

Comment:
```suggestion
                    if (formingDate(item.attributes.time) === chosenTime) {
```
seems to be easier to understand for me

Reply:
It won't work. That fn is for dd-mm-yyyy format and solving timezone problem